corehq/apps/fixtures/tests/test_upload.py

```diff
@@ -29,10 +30,32 @@ class TestFixtureUpload(SimpleTestCase):
 
     def _test(self, config):
         wb = FixtureWorkbook(config.upload_file)
-        with self.assertRaises(FixtureUploadError) as context:
+        if config.error_messages:
+            with self.assertRaises(FixtureUploadError) as context:
+                wb.validate()
+            self.assertEqual(context.exception.errors, config.error_messages)
+        else:
+            # assert doesn't raise anything
             wb.validate()
-        self.assertEqual(context.exception.errors, config.error_messages)
 
+    test_ok = _upload_test('ok', [])
+
+    def test_comprehensiveness(self):
```

Comment:
if you wanted to get even more meta you could add a test for this test. :recycle: